packages/pnpm/test/help.spec.ts

```diff
@@ -8,3 +8,11 @@ test('print an error when help not found', (t) => {
   )
   t.end()
 })
+
+test('print help', (t) => {
+  t.equal(
+    createHelp({})({}, ['foo']).split('\n')[1],
+    '',
```

Comment:
what is this test doing?

Reply:
This was supposed to test the newly added command descriptions. I am trying to figure out how to selectively run this test only , can you help me?

Reply:
you can use `test.only(`